src/components/Table/readme.md

```diff
@@ -981,6 +981,145 @@ function TableExample(props) {
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',  
+        id: '5678zdfgdf',
+    },
+    {
+        name: 'Carlos',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '3434fgfgdf',
+    },
+    {
+        name: 'Luis',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012asdfgh',
+    },
+];
+const Title = styled.h1.attrs(props => props.theme.rainbow)`        
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+    `;  
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+    `;
+
+function TableCustomAction(props) {
+    const [data, setData] = useState(props.data);
+
+    function handleOnDeleteElement(id) {
+            console.log(id)
+            const newData = data.filter(e => e.id!==id);
+            setData(newData)
+        }
+    const customActionStyles ={marginLeft: "auto"};
+    const StyledActionCell = styled.div`
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        align-items: center;
+        margin: auto;
+        box-sizing: border-box;
+    `;
+    const CustomAction = ({row})=>(
+            <StyledActionCell>
+                <ButtonIcon
+                        onClick={()=>handleOnDeleteElement(row.id)}
+                        buttonSize="small"
+                        style={customActionStyles}
+                        icon={<FontAwesomeIcon icon={faTrashAlt} />}
+                        />
+            </StyledActionCell>    
+    );
+    return ( 
+        <div className="rainbow-p-bottom_xx-large"> 
+            <GlobalHeader  src="images/user/user3.jpg">
+                <ButtonGroup className="rainbow-m-right_medium">
+                    <ButtonIcon variant="border-filled" disabled icon={<FontAwesomeIcon icon={faPlus} />} />
+                    <ButtonIcon
+                        variant="border-filled"
+                        disabled
+                        icon={<FontAwesomeIcon icon={faEllipsisV} />}
+                    />
+                </ButtonGroup>
+            </GlobalHeader>
+            <div className="rainbow-flex rainbow-justify_spread rainbow-align_center rainbow-p-around_large">
+                    <div>
+                        <Title>Members</Title>
+                        <Subtitle>Total • {data.length}</Subtitle>
+                    </div>
+                        
+            </div>
+            <Table keyField="id" data={data}>
+                <Column header="Name" field="name" />
+                <Column header="Description" field="description" />
```

Comment:
I think it would be much more useful if we make the text appear complete in the description column, I mean to wrap it. It would be much more evident if we put the text of the description a little longer.
In this way, we would have an example that covers this use case that we use many times.
You can follow the design example

Reply:
@lyon93 With this, I mean that we make the "Description" column have more than one line of text in the example (that is, do not put the ellipsis "...") just like the design
<img width="713" alt="Screen Shot 2020-07-29 at 4 25 48 PM" src="https://user-images.githubusercontent.com/32083556/88855271-387b1700-d1b8-11ea-9087-831f250a4590.png">